### ISSUE TITLE
Being able to change the settings of all Reshade Effects

### ISSUE THREAD

**Issue by Toadfield** [benjamimgois/goverlay#91]
For many Reshade filters you can change the settings of the effects,like how strong they are,but in the current state of GOverlay I can only set if the shader is on or off.

**Comment by Toadfield** [comment]
What I mean is [this](https://i.imgur.com/fJSXSPS.png).
You can edit the settings in the shader files,but it would be cool if you could do it in GOverlay.

**Comment by benjamimgois** [comment]
@Toadfield this is a huge task. since there is so many reshade effects and they are constantly updating. I really don't think i could deal with it alone. But i can make a GUI for the principal ones at least. Or.... i can create a button to open the text file so you can make advanced customization there.

**Comment by Toadfield** [comment]
yeah,that would be great if one of those ways you mentioned would be implemented.
